pkg/option/option.go

```diff
@@ -234,15 +240,15 @@ func ParseOption(arg string, lib *OptionLibrary) (string, bool, error) {
 	return key, enabled, nil
 }
 
-// getFmtOpt returns #define name if option exists and is set to true in endpoint's Opts
+// getFmtOptLocked returns #define name if option exists and is set to true in endpoint's Opts
 // map or #undef name if option does not exist or exists but is set to false
-func (bo *BoolOptions) getFmtOpt(name string) string {
+func (bo *BoolOptions) getFmtOptLocked(name string) string {
```

Comment:
IMO I don't think it's necessary to write "Locked" on private methods.

private methods -> lock already acquired.
public methods -> call from outside so we need to acquire lock.

This is just a thought, what do you think?

Reply:
By this logic private "isEnabledLocked" should then be called "isEnabled", where the locking difference to public "IsEnabled" should be deduced from the name. Not sure if this is what you want?

Reply:
Of maybe you meant that "Locked" suffix is not needed on private methods when there is no corresponding public function?

I'm going by this and doing exactly what you asked for :-)

Reply:
>Of maybe you meant that "Locked" suffix is not needed on private methods when there is no corresponding public function?

Correct